app/views/categories/show.html.erb

```diff
@@ -0,0 +1,43 @@
+<nav class="navbar"> 
+   <div class="hum">
+          <%=image_tag "hum.svg" ,size: "37x37"%>
+        </div>
+
+
+  <span class="or-title cat-title"><%= @category.name%></span>
+
+  </nav> 
+
+
+
+
+
```

Comment:
[Optional] Kindly remove whitespace in your code to make to more professional 👍 

Check for other occurrences